apps/system/utils/netcmd_netmon.c

```diff
@@ -155,17 +186,11 @@ int cmd_netmon(int argc, char **argv)
 	sq_queue_t q_data;
 	sq_init(&q_data);
 	int ret;
-	if (argc > 2) {
-		printf("Bad input\n");
-	} else if (argc == 1) {
-		/* Get default information: SIOCGETSOCK (default) */
-		ret = netlib_netmon_sock(&q_data);
-		if (!ret) {
-			/* Free sq_queue entry of netmon_sock in print_socket() */
-			print_socket(&q_data);
-		} else {
-			printf("Failed to fetch socket info.\n");
-		}
+	char *intf = NULL;
+	struct netmon_netdev_stats stats = {0};
+
+	if (argc == 1 || argc > 2) {
+		print_help();
 	} else if (!(strncmp(argv[1], "sock", strlen("sock")))) {
```

Comment:
strlen("sock") + 1 instead of strlen("sock")

Reply:
Modification completed.